src/containers/Movie/MovieInfo/component.js

```diff
@@ -0,0 +1,141 @@
+import React from 'react'
+import {
+  Row,
+  Col,
+  Typography,
+  Tag
+} from 'antd'
+import PropTypes from 'prop-types'
+import { timeConvert, isoLang } from 'Utils/'
+
+const MovieInfo = ({ info, genres }) => {
+  const {
+    originalTitle,
+    originalLanguage,
+    overview,
+    releaseDate,
+    runtime,
+    budget,
+    revenue,
+    firstAirDate,
+    originalName
+  } = info
+  return (
+    <>
+      <Row>
+        <Col
+          span={20}
+          offset={2}
+        >
+          <Typography.Title>
+            <span>
+              {originalTitle || originalName}
+              {' '}
+              (
+              {new Date(releaseDate || firstAirDate).getFullYear()}
+              )
+            </span>
+          </Typography.Title>
+          <Typography.Title level={3}>Overview</Typography.Title>
+          <Typography.Paragraph>
+            {overview}
+          </Typography.Paragraph>
+        </Col>
+      </Row>
+      <Row>
+        <Col
+          span={20}
+          offset={2}
+        >
+          <Typography.Paragraph>
+            <b>Original Language: </b>
+            <span>{isoLang[originalLanguage]}</span>
+          </Typography.Paragraph>
+        </Col>
+        {Boolean(runtime) && (
+          <Col
+            span={20}
+            offset={2}
+          >
+            <Typography.Paragraph>
+              <b>Runtime: </b>
+              <span>{timeConvert(runtime)}</span>
+            </Typography.Paragraph>
+          </Col>
+        )}
+        {Boolean(budget) && (
+          <Col
+            span={20}
+            offset={2}
+          >
+            <Typography.Paragraph>
+              <b>Budget: </b>
+              {/* eslint-disable-next-line react/jsx-one-expression-per-line */}
+              <span>${budget.toFixed(2)}</span>
+            </Typography.Paragraph>
+          </Col>
+        )}
+        {Boolean(revenue) && (
+          <Col
+            span={20}
+            offset={2}
+          >
+            <Typography.Paragraph>
+              <b>Revenue: </b>
+              {/* eslint-disable-next-line react/jsx-one-expression-per-line */}
+              <span>${revenue.toFixed(2)}</span>
+            </Typography.Paragraph>
+          </Col>
+        )}
+        <Col
+          span={20}
+          offset={2}
+        >
+          <Typography.Paragraph>
+            <b>Genres: </b>
+            {Boolean(genres?.length) && genres.map(genre => (
```

Comment:
`map` from 'lodash` can help to go through the collection without any additional checks